packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -231,7 +231,7 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
                 logCounter: LogUtils.getCounter(),
               })
             } catch (err: any) {
-              const wrappedErr = $errUtils.errByPath('origin.run_domain_fn_errored', {
+              const wrappedErr = $errUtils.errByPath('origin.run_origin_fn_errored', {
```

Comment:
Any reason the `$errUtils.throwErrByPath` can't be used here?

Reply:
This will call errorByPath and then allow the [options to merge with the error being thrown](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/error_utils.ts#L247). Though name migh tneed to be set as [errProps](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/error_utils.ts#L208). Seems possible but wdik??
```js
$errUtils.errByPath('origin.run_domain_fn_errored', {
  args: { error: err.message },
  onFail: () => {},
  errProps: { name: err.name, stack: err.stack },
})
```